.buildbot.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+./x.py clean
+RUST_BACKTRACE=1  ./x.py build --config .buildbot.toml --stage 1 src/rustc
```

Comment:
This is a very minimal build without tests. I assume that's intentional since I doubt your backend can bootstrap yet. Am I right?

Reply:
This is a niggle, but you have a double space on this line also.

Reply:
Yep, I wanted a minimal build, at least for now. I'll remove the space.